app/views/comments/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="mb-5">
+  <div class="card bg-light">
+    <div class="card-body">
+    <!-- Comment form-->
+        <a href="<%= user_path(@user) %>">
```

Comment:
- [ ] This page has error when you load the page to create a comment. Kindly fix it. Take note of the earlier suggestion. You have not implemented the comment create feature.

  ![image](https://github.com/Izebiafe/solo-blog/assets/54848363/d6181b63-258e-4f0b-b7b2-fb8f30125be2)